src/accessories/characteristics/brightness.ts

```diff
@@ -63,7 +63,8 @@ export class BrightnessCharacteristic extends TuyaWebCharacteristic {
     ) {
       stateValue = Number(data.color.brightness);
     } else if (data?.brightness) {
-      stateValue = Math.round((Number(data.brightness) / 255) * 100);
+      const max_brightness = data?.max_brightness ?? 255;
```

Comment:
`max_brightness` should be defined as a getter property on the accessory. It is not part of the data.

Please take a look at the `currentTemperatureFactor` method in the `TemperatureSensorAccessory`.